src/index.js

```diff
@@ -1 +1,5 @@
-import './index.css';
\ No newline at end of file
+import './index.css';
+import LoginScreen from './screens/login/login';
+
+const login = new LoginScreen(document.getElementById('screen'), 'my@email');
+login.render(document.getElementById('screen'), 'hidden');
```

Comment:
Instead of passing string as a second parameter I would suggest passing an object called 'props' or 'options' with any properties that the component needs.